src/globals/index.ts

```diff
@@ -12,6 +12,13 @@ export const code2tokenMap: Code2Name = {
   REP: 'AUGUR',
 }
 
+export const network2URL: Network2URL = {
+  RINKEBY: 'https://rinkeby.etherscan.io/',
+  MAIN: 'https://etherscan.io/',
+  KOVAN: 'https://kovan.etherscan.io/',
+  UNKNOWN: 'localhost:5000/',
```

Comment:
So this will create `localhost:5000/token/address` which leads nowhere? I guess it doesn't really matter
Also do `//localhost:500/` to match protocol on localhost only

Reply:
ya